packages/rainbowkit/src/wallets/Wallet.ts

```diff
@@ -70,23 +70,38 @@ export type Wallet<C extends Connector = Connector> = {
     linux?: string;
     desktop?: string;
   };
-  hidden?: (args: {
-    wallets: {
-      id: string;
-      connector: Connector;
-      installed?: boolean;
-      name: string;
-    }[];
-  }) => boolean;
-  createConnector: () => RainbowKitConnector<C>;
-};
+  hidden?: () => boolean;
+  createConnector: (options: WalletDetailsParams) => CreateConnectorFn;
```

Comment:
`WalletDetailsParams` are used to inject our `Wallet` details into `wagmi`'s connector. All of our `extensions`, `qrCode` and `mobile` data goes into the `CreateConnectorFn`. 

Later wagmi uses all of the `CreateConnectorFn` we've passed and gives back our data later in the react tree 🙌